### PR TITLE
add github security alerts issue create in JIRA

### PR DESCRIPTION
**Pull request by uchinda-sph**

### CODE REVIEW

File: .github/workflows/github-security-alerts-jira.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        description: GitHub Organization Service Account Token
+        required: true
+      JIRA_TOKEN:
+        description: JIRA DevOpsInt User Account Token
+        required: true
+      
+jobs:
+  syncSecurityAlerts:
+    permissions: read-all
+    runs-on: ubuntu-latest
```
Comment (niroz89) [inline .github/workflows/github-security-alerts-jira.yaml]: Let's run it on the enterprise runners.

**Review by niroz89 — COMMENTED** [review]
@uchinda-sph this reusable workflow is tested on tm ?

**Comment by uchinda-sph** [comment]
> @uchinda-sph this reusable workflow is tested on tm ?

yes, 
https://github.com/SPHTech/tm-d9/actions/runs/4750158704/jobs/8438084924
https://github.com/SPHTech/tm-d9/blob/dev/.github/workflows/github-security-jira.yaml

File: .github/workflows/github-security-alerts-jira.yaml (hunk 1 above)
Comment (uchinda-sph) [reply .github/workflows/github-security-alerts-jira.yaml]: > Let's run it on the enterprise runners.

Sure

**Review by niroz89 — APPROVED** [review]

**Pull request merged**